controllers/tweets.controller.js

```diff
@@ -1,5 +1,216 @@
-class TweetsController{
+import Tweet from "../models/tweets.model.js";
+import Like from "../models/likes.model.js";
+import Comment from "../models/comments.model.js";
+import User from "../models/users.model.js";
 
-}
+// Create a new tweet
+const createTweetController = async (req, res) => {
+  try {
+    const { content } = req.body;
 
-module.exports = new TweetsController();
\ No newline at end of file
+    const tweetedUser = await User.findById(req.user._id);
+    const newTweet = await Tweet.create({ content, userId: req.user._id });
+
+    tweetedUser.tweets.push(newTweet._id);
+    await tweetedUser.save();
+    res.status(201).json({
+      message: "Tweet created successfully",
+      tweet: newTweet,
+    });
+  } catch (error) {
+    console.error("Error creating tweet: ", error);
+    res
+      .status(500)
```

Comment:
Handle the error with correct status code and message. 500 should not be thrown.